parser/json/value/type.go

```diff
@@ -8,7 +8,7 @@ const (
 	UnknownType Type = iota
 	NullType
 	BoolType
-	StringType
+	BytesType
```

Comment:
Probably should be `StringType` still for JSON values, and convert them at callsite.